### ISSUE TITLE
'internal error: entered unreachable code: Comment should contain '#' character'

### ISSUE THREAD

**Issue by qarmin** [charliermarsh/ruff#3424]
0.0.254

```
thread 'main' panicked at 'internal error: entered unreachable code: Comment should contain '#' character', crates/ruff/src/rules/eradicate/rules.rs:45:5
stack backtrace:
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
```
[v1_resource_quota_list31.py.zip](https://github.com/charliermarsh/ruff/files/10936011/v1_resource_quota_list31.py.zip)

**Comment by charliermarsh** [comment]
I think that contains a CR line ending which is probably the issue.

**Comment by charliermarsh** [comment]
Resolved via #3454 and #3439.
